### ISSUE TITLE
Accessibility: Edit Address form required indicator fails color contrast

### ISSUE THREAD

**Issue by amberhinds** [woothemes/woocommerce#43634]
### Prerequisites

- [X] I have carried out troubleshooting steps and I believe I have found a bug.
- [X] I have searched for similar bugs in both open and closed issues and cannot find a duplicate.

### Describe the bug

The required indicator on the edit address form uses "red" for its color, which fails color contrast.

https://github.com/woocommerce/woocommerce/blob/758df4854d7b627fc26eb5c0860dc492dea6a06e/plugins/woocommerce/client/legacy/css/woocommerce.scss#L1252

### Expected behavior

All field labels and required indicators should pass color contrast checks at a [minimum of 4.5:1 (WCAG AA)](https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html) but most ideally at a ratio of [7:1 (WCAG AAA)](https://www.w3.org/WAI/WCAG21/Understanding/contrast-enhanced.html), to ensure that people who have low vision can complete checkout.

### Actual behavior

HTML red fails contrast on white and needs to be darkened.

### Steps to reproduce

See linked code above.

### WordPress Environment

N/A

### Isolating the problem

- [X] I have deactivated other plugins and confirmed this bug occurs when only WooCommerce plugin is active.
- [X] This bug happens with a default WordPress theme active, or [Storefront](https://woo.com/products/storefront/).
- [X] I can reproduce this bug consistently using the steps above.
